### ISSUE TITLE
Cannot add View Component in master

### ISSUE THREAD

**Issue by willr3** [Hyperfoil/Horreum#1294]
**Describe the bug**

Trying to add a view component in a build of master will show a `Sign in` popup that does not accept my credentials and prevents the view from being updated.

![Screenshot from 2024-02-07 22-48-35](https://github.com/Hyperfoil/Horreum/assets/1083859/503221b9-3af3-4b68-9ac6-a8ebe2166e64)

**To Reproduce**

Add a View Component to a Test you own and click `Save`. 
I was able to reproduce the problem with both the `Default` view and a new View.

**Version**

What is the version of Horreum ? master branch

If you are using a development branch; what is the commit id ? 

```
git rev-parse HEAD
fa9dbfea0cad695c1e61f6f3cd4bc7c099bbc466
```
**Java**

What is the version of Java ?

```
java -version
openjdk version "17.0.10" 2024-01-16
OpenJDK Runtime Environment Temurin-17.0.10+7 (build 17.0.10+7)
OpenJDK 64-Bit Server VM Temurin-17.0.10+7 (build 17.0.10+7, mixed mode, sharing)

```

**Comment by willr3** [comment]
I am running the dev instance with a production database backup and I also cannot see anything in the Administration views

**Comment by willr3** [comment]
fixed in afee7262ec32ffed88279c692de239247329f7d4
